mtp_api/apps/service/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-09-21 15:11
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Downtime',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('service', models.CharField(choices=[('gov_uk_pay', 'GOV.UK Pay')], max_length=50)),
+                ('start', models.DateTimeField(blank=True, null=True)),
```

Comment:
is this migration outdated? `blank=True, null=True` looks like it should be different…